src/Microsoft.OData.Core/JsonLight/ODataJsonLightContextUriParser.cs

```diff
@@ -327,7 +327,7 @@ private ODataPayloadKind ParseContextUriFragment(string fragment, Func<IEdmType,
                 {
                     detectedPayloadKind = ODataPayloadKind.ServiceDocument;
                 }
-                else if (fragment.Equals(ODataConstants.CollectionPrefix + "(" + ODataConstants.EntityReferenceSegmentName + ")", StringComparison.Ordinal))
+                else if (fragment.Equals(string.Concat(ODataConstants.CollectionPrefix, "(", ODataConstants.EntityReferenceSegmentName, ")"), StringComparison.Ordinal))

```

Comment:
>else if (fragment.Equals(string.Concat(ODataConstants.CollectionPrefix, "(", ODataConstants.EntityReferenceSegmentName, ")"), StringComparison.Ordinal)) [](start = 16, length = 152)

Since these are all static values, it probably makes sense to define an ODataConstant.EntityReferenceCollection = string.Concat(ODataConstants.CollectionPrefix, "(", ODataConstants.EntityReferenceSegmentName, ")")